repository/src/main/java/org/apache/atlas/repository/store/graph/v1/RestoreHandlerV1.java

```diff
@@ -567,4 +574,18 @@ private void restoreAllClassifications(AtlasVertex instanceVertex) throws AtlasB
             restoreEdgeReference(edge, CLASSIFICATION, false, instanceVertex);
         }
     }
+
+    public boolean isRelationshipEdge(AtlasEdge edge) {
```

Comment:
I think this method is not used, can be removed.

Reply:
No, This method is what fixes the issue
I added this method

Reply:
I think I am missing something here, not able to find caller of this method.